src/graph/RadarPostprocessPointsNode.cpp

```diff
@@ -74,29 +88,32 @@ void RadarPostprocessPointsNode::enqueueExecImpl()
 
 	distanceInputHost->copyFrom(input->getFieldData(DISTANCE_F32));
 	azimuthInputHost->copyFrom(input->getFieldData(AZIMUTH_F32));
+	speedInputHost->copyFrom(input->getFieldData(RADIAL_SPEED_F32));
 	elevationInputHost->copyFrom(input->getFieldData(ELEVATION_F32));
 
 	std::vector<RadarCluster> clusters;
 	// Create first cluster with the first point
 	clusters.emplace_back(0, distanceInputHost->getReadPtr()[0], azimuthInputHost->getReadPtr()[0],
-	                      elevationInputHost->getReadPtr()[0]);
+	                      speedInputHost->getReadPtr()[0], elevationInputHost->getReadPtr()[0]);
 
 	for (int i = 1; i < input->getPointCount(); ++i) {
 		auto distance = distanceInputHost->getReadPtr()[i];
 		auto azimuth = azimuthInputHost->getReadPtr()[i];
+		auto speed = speedInputHost->getReadPtr()[i];
```

Comment:
```suggestion
		auto radialSpeed = speedInputHost->getReadPtr()[i];
```
Also, all of these, except the boolean flag, can be made const.